german/the-black-hack.md

```diff
@@ -46,9 +46,9 @@ Rüstung bietet Schutz, indem sie allen erlittenen Schaden reduziert. Jedes Stü
 
 |     Typ        | Punkte |
 |:--------------:|:------:|
-|  Festes Wams   |   2    |
-|  Lederrüstung  |   4    |
-|   Kettenhemd   |   6    |
+| Gambeson       |   2    |
```

Comment:
" Ich lasse mich da gern eines Besseren belehren (Eintrag im Wörterbuch, Verbreitung des Wortes im Internet etc.)"
Anscheinend ja nicht, weil das Verbreitung im Internet Argument samt Wikipedia ja wohl nicht verbreitet genug ist um dem anfänglichen Argument Neueinsteigern keine In-Group Worte vorzulegen mit denen sie nix anfangen können, auch nur wie ein Vorwand rüber kommt um ja nichts wie bei DSA zu benennen. Da es nicht nur in DSA vorkommt wird, sondern auch außerhalb verwendet wird ignorierst du völlig und wirfst immer nur dein DSA Schild vor ohne auch nur einen anderen Grund zu nennen oder DSA mal völlig außen vor zu lassen.
Dann bringen wir es mal auf den Punkt, den ich eben gemacht habe: Die Google Suche. Wo kommt für einen Neuling verwertbarere Informationen heraus? Wenn man nach Festes Wams sucht oder nach Gambeson.
Mir geht es darum, dass der Spieler oder SL, der mit solchen Begriffen nichts anfangen kann auch direkt die richtige Erklärung findet. Die liefert Google nunmal über Gambeson. Bei allem anderen muss derjenige viel mehr Sucharbeit investieren und das geht für mich über Einsteigerfreundlich hinaus.
Und können wir bitte DSA komplett aus der Argumentationskette lassen? Ich versuche das ja auch zu meiden wo es nur geht.^^